src/Altinn.AccessManagement/Controllers/MaskinportenSchemaController.cs

```diff
@@ -91,7 +94,64 @@ public async Task<ActionResult<List<MPDelegationExternal>>> GetMaskinportenDeleg
                 return new ObjectResult(ProblemDetailsFactory.CreateProblemDetails(HttpContext));
             }
         }
-  
+
+        /// <summary>
+        /// Endpoint for performing a query of what rights a user can delegate to others on behalf of a specified reportee and maskinporten schema.
+        /// </summary>
+        /// <param name="party">The reportee party</param>
+        /// <param name="rightsDelegationCheckRequest">Request model for user rights delegation check</param>
+        /// <response code="200" cref="List{RightDelegationStatusExternal}">Ok</response>
+        /// <response code="400">Bad Request</response>
+        /// <response code="401">Unauthorized</response>
+        /// <response code="403">Forbidden</response>
+        /// <response code="500">Internal Server Error</response>
+        [HttpPost]
+        [Authorize(Policy = AuthzConstants.POLICY_MASKINPORTEN_DELEGATION_WRITE)]
+        [Route("{party}/maskinportenschema/delegationcheck")]
+        [ApiExplorerSettings(IgnoreApi = false)]
```

Comment:
can remove the ApiExplorerSettings attribute when it's not to be hidden from the swagger.